src/lib/git/azure.ts

```diff
@@ -22,6 +26,15 @@ let gitApi: IGitApi | undefined; // keep track of the gitApi so it can be reused
 // Helpers
 ////////////////////////////////////////////////////////////////////////////////
 
+/**
+ * Returns azure organization URL.
+ *
+ * @param orgName Organization name
+ */
+export const getAzureOrganizationUrl = (orgName: string): string => {
+  return `https://dev.azure.com/${orgName}`;
```

Comment:
it's possible for an organization to be hosted on a visualstudio.com domain ie: `https://${organization}.visualstudio.com/`

Reply:
@andre what is your take on this? Because currently bedrock setup only support azure dev ops

Reply:
Yes, some folks are using `https://${organization}.visualstudio.com`. @bnookala are you aware of an API to check this?

This isn't something we are going to address in this PR. Feel free to make an issue for it. We don't need to block this PR for that.

Reply:
i'm not aware of any api check - we've looked at the git remote to extrapolate what url scheme the url falls under in the past (see: https://github.com/microsoft/bedrock-cli/blob/60c89e1d7c09ae55bb622359baa3a6bbba2824a0/src/lib/gitutils.ts#L236) - maybe you can do a similar check based on the input git urls. 

for the sake of completeness, i would ask that you have this fixed in this PR, as we have had to do bug fixes in the past to fix this issue for users - and have a learning that this is something that users _will_ run into if they use visualstudio.com domains with the setup command.

Reply:
all I have is an org name for azure devops. can you suggest what i need to this function? thanks

Reply:
@dennisseah okay, just looked and seems like setup assumes strict dev.azure.com domains - i'll resolve this for now, and file a ticket to make it support visualstudio.com domains

Reply:
ok, thanks

Reply:
@dennisseah @andrebriggs  https://github.com/microsoft/bedrock/issues/1348

Reply:
@bnookala can you unblock this PR?